cinch/roles/jenkins_master/templates/config.xml.j2

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Templating this entire config file will require a bit of auditing, in that there are many settings we are configuring here.  Some of these settings may exist in this project as variables ( e.g. <useSecurity>), so we really need to audit the rest of the cinch code to make sure that putting this entire template in place doesn't break other settings.  We also need to be sure later things we're doing don't interfere with this config file and break idempotence.